docs/conf/prague/2019/convince-your-manager.rst

```diff
@@ -0,0 +1,65 @@
+:template: {{year}}/generic.html
+
+Convince Your Manager
+=====================
+
+Do you need help justifying why your employer should send you to Write the Docs? Don't worry – you're not alone.
+Based on the experiences of some of our previous attendees, we've put together a sample email and list of resources below.
+Feel free to adapt and share with your manager to show them the many benefits of attending!
+
+Sample email
+-------------
+
+Remember to change the things in `[brackets]`!
+
+----
+
+  FROM: [your name]
+
+  TO: [your employer or manager's name]
+
+  SUBJECT: Professional Development: Documentation Community Conference
+
+  I'd like to attend Write the Docs {{city}}, which takes place {{ date.main }}. This three-day event explores the art and science of documentation, and covers a diverse range of topics related to documentation in the software industry.
+
+  Write the Docs conferences bring together everyone who writes the docs – Tech Writers, Developers, Developer Relations, Customer Support – making the events an ideal networking opportunity.
+  Each conference successfully combines a number of different event formats to deliver engaging, practical, and timely content.
+
+  There is a single track of talks, a parallel unconference event, and a community writing day. The `sessions from last year </conf/eu/{{year-1}}/speakers/>`_ will give you a good idea of the kinds of topics covered, many of which are relevant to my work.
+
+  Costs:
+
+  * Conference ticket (includes breakfast and lunch) - {{tickets.corporate.price}}
+  * Travel – [fill in with estimate]
+  * Accommodation – – [fill in with estimate]
+
+  Benefits:
+
+  * Discovering solutions to problems I'm facing at work
+  * Exposure to the latest ideas, techniques, and tools for software docs
+  * Opportunity to learn from the best doc teams in the industry
+  * Building professional connections with other documentarians
+
+  Thanks in advance,
+  [your name]
+
+----
+
+Resources
+---------
+
+When discussing how to pitch the conference, a few generally helpful tips emerged:
+
+* Highlight a few specific talks that relate to ongoing projects at work. (This one's dependent on pitching after the talk line up has been announced).
+* If your company is looking to hire another documentarian, the job fair and networking at the event are an excellent resource.
+* Don't forget that one of the benefits to your attendance is that it raises the visibility of your company in the community. If your team wants a reputation for caring about their docs, having people at Write the Docs is a great way to do that.
+
+In addition, it can be useful to share some info about previous conferences. You can find the websites for previous events on :doc:`/conf/index/`, and a quick list of last year's talks down below.
+But perhaps more useful might be some of the info in our :doc:`press-kit/`, which includes community testimonials, photos, and more.
+
+List of talks from 2017
```

Comment:
Wonder if we can `{{year-1}}` all of these 2017s to make them 2018

Reply:
Yes, we can. With some filename changes that happened in between.